docs/topics/testing/tools.txt

```diff
@@ -1605,6 +1605,45 @@ your test suite.
             Person.objects.create(name="Aaron")
             Person.objects.create(name="Daniel")
 
+.. _topics-tagging-tests:
+
+Tagging tests
+-------------
+
+.. versionadded:: 1.9
+
+You can tag your tests to group them- for example fast and slow tests::
```

Comment:
You can tag your tests to group them. For example, you might label fast or slow tests::